src/en/reference-bundle.md

```diff
@@ -0,0 +1,346 @@
+Title: Bundle reference
+
+# Bundle reference
+
+A bundle is defined by a [YAML-formatted][yaml] file with a fixed set of
+elements. This document will define the properties allowed in such a file and
+give the format of expressing them by way of examples.
+
+A bundle file makes available a number of charm series settings. A section on
+how a charm's series is ultimately determined is therefore presented first.
+
+Page [Charm bundles][charms-bundles] gives an overview of bundle usage.
+
+## Charm series
+
+What series a charm will use can be influenced in several ways. Some of these
+are set within the bundle file while some are not. When using bundles, the
+series is determined using rules of precedence (most preferred to least):
+
+ - the series stated in a charm URL (see `charm` under the
```

Comment:
We are running away as fast as possible from using charm urls with series embedded, preferring multi-series charms (with no series in the url). I'd rather not see us document a feature we are trying to get rid of.

Reply:
Gotcha. I may get around to updating the help for `deploy` to make things consistent.